dm/tests/incremental_mode/data/db1.prepare.sql

```diff
@@ -1,5 +1,4 @@
 drop database if exists `incremental_mode`;
-reset master;
```

Comment:
please add a clear message in this files, so we won't add reset master later mistakenly.

also please add a clear comments in run.sh that it's used in chaos test 99999, something like:
```
############################
#            xxxxxxxxxxxxxxxxxxx
############################
```